domain/error/error_domain_v3.0.0.yaml

```diff
@@ -119,14 +119,14 @@ components:
           example: '4426d965-0dd8-4005-8c63-dc68b01c4962'
         errorDateTime:
           description: |
-            The DateTime corresponding to the error occuring. Must be formatted using [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) format.
+            The DateTime corresponding to the error occurring. Must be formatted using [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) format.
```

Comment:
Unfortunately we cannot modify this Domain. Can you please make a new `error_domain_v3.0.1.yaml` with *only* these typos fixed? (And still keep the 4.0.0 file where you also fix the property name)